src/Razor/src/Microsoft.AspNetCore.Razor.Common/Microsoft.AspNetCore.Razor.Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework);netstandard2.0;$(DefaultNetFxTargetFramework)</TargetFrameworks>
```

Comment:
Should `netstandard2.0` also be a property in case Roslyn OOP upgrades?

Reply:
Possibly. Although, I'm not really a fan of these MSBuild properties for TFMs. This was just an attempt to at least use them consistently. Having an MSBuild property for `netstandard2.0` at this point seems a bit contrived to me.